docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -201,6 +201,10 @@ Sub-sections related to the Administration Server, specific clusters, or specifi
 
 The elements `serverStartPolicy`, `serverStartState`, `serverPod` and `serverService` are repeated under `adminServer` and under each entry of `clusters` or `managedServers`.  The values directly under `spec`, set the defaults for the entire domain.  The values under a specific entry under `clusters`, set the defaults for cluster members of that cluster.  The values under `adminServer` or an entry under `managedServers`, set the values for that specific server.  Values from the domain scope and values from the cluster (for cluster members) are merged with or overridden by the setting for the specific server depending on the element.  See [Startup and shutdown]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md" >}}) for details about `serverStartPolicy` combinations.
 
+Elements related to customization of liveness probe:
+
+* `livenessProbeCustomScript`: Full path of the script to customize the liveness probe for WebLogic server instance pods. The generated liveness probe script `livenessProbe.sh` will invoke the custom script provided by this element and if the custom script fails with non-zero exit status, pod will fail the liveness probe and Kubernetes will restart the container.
```

Comment:
similar to the arch document, make it clear thte the custom script is _in addition to_ the current script, and mention how to fine tune the interval aand timeout

Reply:
Also make it clear that this is optional, for advanced usage only (rarely needed), what the default is, and what happens if the specified script is not found...